### PR TITLE
PRL blog

### PR DESCRIPTION
**Pull request by bennn**
Initial draft for PRL blog.
~~@wilbowma  when I click post titles, it takes me to `/blog` instead of the post's page. Can you figure out what's wrong?~~ (edit: done)

~~I think @maxsnew (or was it @takikawa ?) offered to write the first post. Let's include that in the pull request before merging.~~

edit: we'll just have the intro for now. I think post `#`2 will be Gabriel announcing NEPLS

### CODE REVIEW

**Comment by wilbowma** [comment]
> @wilbowma  when I click post titles, it takes me to `/blog` instead of the post's page. Can you figure out what's wrong?
> Will do.

**Comment by schuster** [comment]
FYI, I have a post about PLAI on my site (http://jschuster.org/blog/) that I'd happily cross-post, and another in the works about lifetime parameters in Rust.

**Comment by bennn** [comment]
Hey everybody, I just drafted a "welcome" post. Please take a look & let's release the blog within the next 2 weeks.

File: blog/_src/posts/2016-04-29-welcome-to-the-prl-blog.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+An open pull request is the best place to ask questions about the formatting
+or content of a post.
+We promise that within a few days of opening a PR someone with push access will
+reply with feedback or merge the request.
+
+Contributions are open to anyone: current labmates, alumni, friends from the
+ Racket mailing list, and even recovering C programmers.
+One should have a strong connection to Northeastern or our research, but
+ even that is not strictly necessary.
+Visitors are always welcome the the PRL.
```
Comment (takikawa) [inline blog/_src/posts/2016-04-29-welcome-to-the-prl-blog.md]: Typo "the" -> "to"

**Comment by takikawa** [comment]
Nice, looks good to me.

**Comment by bennn** [comment]
ok I'm going to push this 📦

**Pull request merged**